### PR TITLE
Spark chapter

### PR DESCRIPTION
**Pull request by skars82**
Addded chapter about Spark:

- started to use embedded notebooks (new feature of quarto) with rendered output
- turned of nbstripout
- _freeze-dir contains the output of notebooks
- added desc in index.qmd

### CODE REVIEW

File: dapla-manual/index.qmd [hunk 1]
```diff
@@ -36,4 +36,32 @@ Denne nettsiden er åpen og hvem som helst kan lese det som er skrevet her. Hold
 1. Åpne en PR
 1. Be noen å gjennomgå endringen og eventuelt godkjenne den
 1. Merge PRen
-1. Etter et par minutter skal din endring være synlig! 
+1. Etter et par minutter skal din endring være synlig!
+
+### Embedded notebooks
```
Comment (mmwinther) [inline dapla-manual/index.qmd]: This is starting to get too long and detailed for a front page, I think we should move all these instructions to their own page, something like `contributing.qmd`.

File: dapla-manual/spark.qmd [hunk 2]
```diff
@@ -1 +1,94 @@
...
+
+
+### SparkR
+
+SparkR er et R-grensesnitt for Apache Spark. Det er en R-pakke som gir tilgang til Spark-funksjonalitet fra R. Det er enkelt å bruke, og har mange av de samme funksjonene som dplyr. Se eksempel i notebook under:
+
+{{< embed notebooks/spark/sparkr-intro.ipynb#print-cell echo=true >}}
+
+
+## Lakehouse-arkitektur
```
Comment (mmwinther) [inline dapla-manual/spark.qmd:75]: Is it necessary to include this? Dapla doesn't offer this functionality so it's confusing to have it in the manual.

**Review by mmwinther — COMMENTED** [review]
Very well written and looks good rendered, I have a couple of comments though 😇

**Comment by skars82** [comment]
Made the agreed changes. Kept Lakehousepart and moved contribution-part into a separate chapter.

**Comment by skars82** [comment]
Could you have a look at the changes, @mmwinther?

**Review by mmwinther — APPROVED** [review]
Nice 👍

**Pull request merged**